serde-generate/src/rust.rs

```diff
@@ -167,7 +187,7 @@ where
             }
         }
         writeln!(self.out)?;
-        if !self.with_serialization && !external_names.contains("Bytes") {
+        if !self.config.inner.serialization && !external_names.contains("Bytes") {
```

Comment:
Whats with the exclusion with the Bytes type?

Reply:
This is a hack to allow overriding the definition of `Bytes`. There is no universally good choice unfortunately between `Bytes = Vec<u8>` (doesn't call the `bytes` callback in Serde) and `Bytes = serde_buf::ByteBuf` (creates one user-visible level of indirection). By default, we use the first choice when `serialization` is false and the second otherwise.

Reply:
Oh right! I forgot about that bit :/